packages/zowe-explorer/src/dataset/actions.ts

```diff
@@ -1266,8 +1288,13 @@ export async function pasteMember(
             const inputBoxOptions: vscode.InputBoxOptions = {
                 value: beforeMemberName,
                 placeHolder: localize("pasteMember.inputBox.placeHolder", "Name of Data Set Member"),
+                validateInput: (text) => {
+                    return dsUtils.validateDataSetName(text) === true
+                        ? null
+                        : localize("dataset.validation", "Enter valid dataset name");
```

Comment:
Should this say "Enter valid member name" instead, and perform different validation than for data sets?